### ISSUE TITLE
NewDot on desktop will not load after the site went down

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#8589]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Expensify servers go down
2. I refresh the app and get sent to login
3. Expensify servers come back online
4. I log in
5. I get this screen and refreshing does not help

## Expected Result:
App opens without any issue 

## Actual Result:
NewDot on desktop will not load after the site went down

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Desktop App

**Version Number:** 1.1.52-0
**Reproducible in staging?:** needs repro
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
<img width="1200" alt="2022-03-30_16-57-58" src="https://user-images.githubusercontent.com/43995119/162670272-52e14dd2-dd4e-4e22-b482-2382d3a2523b.png">

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @stitesExpensify 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1648681351033189

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MonilBhavsar** [comment]
Discussing in slack https://expensify.slack.com/archives/C01GTK53T8Q/p1648681351033189

**Comment by MonilBhavsar** [comment]
On hold for offline support/redesign

**Comment by MonilBhavsar** [comment]
Still on hold

**Comment by MonilBhavsar** [comment]
Same ^

**Comment by MonilBhavsar** [comment]
Will be looking at this after reviewing doc, probably next week

**Comment by MonilBhavsar** [comment]
Working this week

**Comment by techievivek** [comment]
Not overdue looking into it today.

**Comment by techievivek** [comment]
Not sure if this is already being handled with offline support. @mvtglobally can you please re-test this. Thanks

**Comment by techievivek** [comment]
Will try to reproduce this locally today.

**Comment by MonilBhavsar** [comment]
I think we should keep it on HOLD until API refactoring is done

**Comment by techievivek** [comment]
Not overdue, will look into it once API refactor stuff is done.

**Comment by techievivek** [comment]
Not overdue, will look at it once the API refactor stuffs are done.

**Comment by techievivek** [comment]
Not overdue.

**Comment by techievivek** [comment]
On hold, will look into it when refactor things are sorted.

**Comment by techievivek** [comment]
Hold on to API refactors.

**Comment by techievivek** [comment]
Hold on to API refactors.

**Comment by techievivek** [comment]
Let me see if it can be worked this week.

**Comment by techievivek** [comment]
On hold for now.

**Comment by techievivek** [comment]
@mvtglobally Can you try reproducing this somehow so we can know if the problem still exists because most probably this would have already been fixed with offline first and API refactor changes?

**Comment by techievivek** [comment]
Not overdue, @mvtglobally can you please have a look at the above comment?

**Comment by techievivek** [comment]
Will try reproducing this once I have some time on my hand.

**Comment by techievivek** [comment]
@mvtglobally Can we please test this once again? Because my guess is it would have been fixed with offline and refactor changes.

**Comment by techievivek** [comment]
Not overdue. I will test this myself tomorrow.

**Comment by techievivek** [comment]
Sorry couldn't look into it, I am working on few App issues so I will be able to test this.

**Comment by techievivek** [comment]
This issue seems to have gotten fixed with offline support. I just halted my Vagrant, and when refreshed the App, it loads data from Onyx. It does fail with `502` for every command send to backend. But I was not redirected to login page.

<img width="1726" alt="Screenshot 2022-11-28 at 6 34 46 PM" src="https://user-images.githubusercontent.com/35863227/204285356-6c670888-011f-48e8-b4d8-73f822b92449.png">

**Comment by techievivek** [comment]
@MonilBhavsar Any thoughts on this one?

**Comment by techievivek** [comment]
Going to close this one because we are no longer logged out automatically when the site goes down. If we need any other specific case to be handled feel free to open a new issue but for the scope of the issue it is already solved by offline support.
